src/DataXml/Utd820/CommonXmlNodes.cs

```diff
@@ -0,0 +1,705 @@
+
+		/// <remarks/>
+		[System.Xml.Serialization.XmlAttributeAttribute()]
+		public ExtendedSignerDetails_BuyerTitle820SignerPowers SignerPowers
+		{
+			get { return this.signerPowersField; }
+			set { this.signerPowersField = value; }
+		}
+
+		/// <remarks/>
+		[System.Xml.Serialization.XmlAttributeAttribute()]
+		public ExtendedSignerDetails_BuyerTitle820SignerStatus SignerStatus
+		{
+			get { return this.signerStatusField; }
+			set { this.signerStatusField = value; }
+		}
+	}
+
+	/// <remarks/>
+	[System.CodeDom.Compiler.GeneratedCodeAttribute("xsd", "4.6.1055.0")]
+	[System.SerializableAttribute()]
+	[System.Xml.Serialization.XmlTypeAttribute(AnonymousType = true)]
+	public enum ExtendedSignerDetails_BuyerTitle820SignerPowers
+	{
+		/// <remarks/>
+		[System.Xml.Serialization.XmlEnumAttribute("1")]
+		Item1,
+
+		/// <remarks/>
+		[System.Xml.Serialization.XmlEnumAttribute("2")]
+		Item2,
+
+		/// <remarks/>
+		[System.Xml.Serialization.XmlEnumAttribute("3")]
+		Item3,
+	}
+
+	/// <remarks/>
+	[System.CodeDom.Compiler.GeneratedCodeAttribute("xsd", "4.6.1055.0")]
+	[System.SerializableAttribute()]
+	[System.Xml.Serialization.XmlTypeAttribute(AnonymousType = true)]
+	public enum ExtendedSignerDetails_BuyerTitle820SignerStatus
+	{
+		/// <remarks/>
+		[System.Xml.Serialization.XmlEnumAttribute("3")]
+		Item3,
+
+		/// <remarks/>
+		[System.Xml.Serialization.XmlEnumAttribute("4")]
+		Item4,
+
+		/// <remarks/>
+		[System.Xml.Serialization.XmlEnumAttribute("5")]
+		Item5,
+
+		/// <remarks/>
+		[System.Xml.Serialization.XmlEnumAttribute("6")]
+		Item6,
+	}
+
+	/// <remarks/>
+	[System.Xml.Serialization.XmlIncludeAttribute(typeof(ExtendedSignerDetails_551_552))]
+	[System.Xml.Serialization.XmlIncludeAttribute(typeof(ExtendedSignerDetails_CorrectionBuyerTitle))]
+	[System.Xml.Serialization.XmlIncludeAttribute(typeof(ExtendedSignerDetails_CorrectionSellerTitle))]
+	[System.Xml.Serialization.XmlIncludeAttribute(typeof(ExtendedSignerDetails_BuyerTitle))]
+	[System.Xml.Serialization.XmlIncludeAttribute(typeof(ExtendedSignerDetails_SellerTitle))]
+	[System.CodeDom.Compiler.GeneratedCodeAttribute("xsd", "4.6.1055.0")]
+	[System.SerializableAttribute()]
+	[System.Diagnostics.DebuggerStepThroughAttribute()]
+	[System.ComponentModel.DesignerCategoryAttribute("code")]
+	public partial class ExtendedSignerDetails : ExtendedSignerDetailsBase
+	{
+		private ExtendedSignerDetailsSignerStatus signerStatusField;
+
+		/// <remarks/>
+		[System.Xml.Serialization.XmlAttributeAttribute()]
+		public ExtendedSignerDetailsSignerStatus SignerStatus
+		{
+			get { return this.signerStatusField; }
+			set { this.signerStatusField = value; }
+		}
+	}
+
+	/// <remarks/>
+	[System.CodeDom.Compiler.GeneratedCodeAttribute("xsd", "4.6.1055.0")]
+	[System.SerializableAttribute()]
+	[System.Xml.Serialization.XmlTypeAttribute(AnonymousType = true)]
+	public enum ExtendedSignerDetailsSignerStatus
+	{
+		/// <remarks/>
+		[System.Xml.Serialization.XmlEnumAttribute("1")]
+		Item1,
```

Comment:
Will you rename this enum values to add more readability?

Reply:
Made more readable values